.travis.yml

```diff
@@ -1,13 +1,62 @@
 language: php
 
+services:
+  - mysql
+
 php:
-  - 5.4
-  - 5.5
   - 5.6
-  - hhvm
+  - 7.0
+  - 7.1
+  - 7.2
+  - 7.3
+  - 7.4
+
+env:
+  - LARAVEL_VERSION=5.6.*
+  - LARAVEL_VERSION=5.7.*
+  - LARAVEL_VERSION=5.8.*
+  - LARAVEL_VERSION=6.*
+
+matrix:
+  exclude:
+    - php: 7.3
+      env: LARAVEL_VERSION=5.6.*
+    - php: 7.4
+      env: LARAVEL_VERSION=5.6.*
+    - php: 7.3
+      env: LARAVEL_VERSION=5.7.*
+    - php: 7.4
+      env: LARAVEL_VERSION=5.7.*
```

Comment:
I don't understand why we ignore PHP 7.3-7.4 for versions of Laravel which support them.

Reply:
to reduce the number of jobs I need to wait now before seeing a failure